config/prod/prow/jobs/custom/test-infra.yaml

```diff
@@ -347,10 +347,16 @@ periodics:
       - name: prow-auto-bumper-github-token
         mountPath: /etc/prow-auto-bumper-github-token
         readOnly: true
+      - name: ssh
```

Comment:
Should this be passed in as an arg?

Reply:
No. It just needs to be in /root/.ssh when the autobumper is run